n_t/msmc.py

```diff
@@ -22,14 +22,16 @@ def write_msmc_file(path):
     for var in ts.variants():
         cur = int(var.site.position)
         if cur > prev:
-            string = chrom+'\t'+str(cur)+'\t'+str(cur-prev)+'\t'+''.join(map(str,var.genotypes))+"\n"
-            fi.write(string)
+            #string = chrom+'\t'+str(cur)+'\t'+ str(cur-prev)
```

Comment:
No point in keeping this lying around, delete.

Reply:
Gotcha; I have more comments that need deleted too. I'll squash and push once I finish these changes for Andy; Should be soon.